apps/web/app/page.tsx

```diff
@@ -1,16 +1,24 @@
 // 3rd party
 import { JSX } from "react";
 // repo
-import Link from "@repo/ui/link";
 import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
-import ModalSide from "@repo/ui/modal-side";
+import WalkthroughCard from "@repo/ui/walkthrough-card";
+import { WalkthroughJSONData } from "../../../packages/data/src/useWalkthroughData";
 
 export default function Page(): JSX.Element {
   return (
     <div className="u-container">
-      <p>
-        <Link href={`${URL_WALKTHROUGH_HREF}/9.9.9/`}>Walkthrough 9.9.9</Link>
-      </p>
+      <div>
+        {Object.entries(WalkthroughJSONData).map(([id, { info }]) => {
+          return (
+            <WalkthroughCard
```

Comment:
These need a key property (ignore if you'd planned to cover that when building the actual page)